pages/services/prometheus/0.1.0-2.3.1/install/install.md

```diff
@@ -0,0 +1,299 @@
+---
+post_title: Install and Customize
+menu_order: 20
+enterprise: 'no'
+---
+
+ DCOS Prometheus is available in the Universe and can be installed by using either the web interface or the DC/OS CLI.
+
+The default DC/OS Prometheus Service installation provides reasonable defaults for trying out the service, but that may not be sufficient for production use. You may require different configurations depending on the context of the deployment.
+
+
+## Configuration Best Practices for Production
```

Comment:
How are these configuration best practices for the framework? this sections seems to talk about `Prometheus Federation` which is a pattern of deployment. This section should contain information of production configuration recommendations.

Reply:
Updated ,building locally and will raise new PR , post making all the change